UsbIpServer/Program.cs

```diff
@@ -117,97 +121,149 @@ void DefaultCmdLine(CommandLineApplication cmd)
                 cmd.Description = "Bind device";
                 var busId = cmd.Option("-b|--busid=<busid>", "Share device having <busid>", CommandOptionType.SingleValue);
                 var bindAll = cmd.Option("-a|--all", "Share all devices.", CommandOptionType.NoValue);
+                var temporary = cmd.Option("-t|--temporary", "Do not automatically share this device if it is reconnected.", CommandOptionType.NoValue);
```

Comment:
You're right. I introduced the `--temporary` flag before I added the polling loop, but in hindsight I think it's overengineered now and I'm happy to remove it. For users that only interact with usbipd-win through the `wsl` command, I don't think they'll notice the difference because they never use raw `usbip` commands on the Linux side, and if they do use the raw commands the persistent behavior is probably better anyway.